megamek/i18n/megamek/common/messages.properties

```diff
@@ -228,10 +228,10 @@ PlanetaryConditions.DisplayableName.Weather.Moderate\ Snowfall=Moderate Snowfall
 PlanetaryConditions.DisplayableName.Weather.Heavy\ Snowfall=Heavy Snowfall
 PlanetaryConditions.DisplayableName.Weather.Sleet=Sleet
 PlanetaryConditions.DisplayableName.Weather.Snow\ Flurries=Snow Flurries
-PlanetaryConditions.DisplayableName.Weather.Blizzard=Blizzard
 PlanetaryConditions.DisplayableName.Weather.Ice\ Storm=Ice Storm
 PlanetaryConditions.DisplayableName.Weather.Light\ Hail=Light Hail
 PlanetaryConditions.DisplayableName.Weather.Heavy\ Hail=Heavy Hail
+PlanetaryConditions.DisplayableName.Weather.Lightning\ Storm=Lightning Storm
```

Comment:
I suggest not using spaces in the name of  the resource if that is not too much effort

Reply:
I see that it is a legacy issue, just a suggestion